src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_TRY = 3;
```

Comment:
Goob, but consider another naming
```suggestion
    private static final int BALLS_COUNT = 3;
```